tests/fast_tests/test_examples.py

```diff
@@ -1,180 +1,148 @@
-
-        # test the example in the presence of inflows
-        exp = traffic_light_grid_example(render=False, use_inflows=True)
-        exp.run(1, 5)
+        """Verify that examples/exp_configs/non_rl/traffic_light_grid.py is working."""
+        self.run_simulation(non_rl_traffic_light_grid)
 
     def test_highway(self):
-        """Verifies that examples/sumo/highway.py is working."""
+        """Verify that examples/exp_configs/non_rl/highway.py is working."""
         # import the experiment variable from the example
-        exp = highway_example(render=False)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
+        self.run_simulation(non_rl_highway)
 
     def test_highway_ramps(self):
-        """Verifies that examples/sumo/highway_ramps.py is working."""
-        # import the experiment variable from the example
-        exp = highway_ramps_example(render=False)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
+        """Verify that examples/exp_configs/non_rl/highway_ramps.py is working."""
+        self.run_simulation(non_rl_highway_ramps)
 
     def test_merge(self):
-        """Verifies that examples/sumo/merge.py is working."""
-        # import the experiment variable from the example
-        exp = merge_example(render=False)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
+        """Verify that examples/exp_configs/non_rl/merge.py is working."""
+        self.run_simulation(non_rl_merge)
 
-    def test_sugiyama(self):
-        """Verifies that examples/sumo/sugiyama.py is working."""
-        # import the experiment variable from the example
-        exp = sugiyama_example(render=False)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
+    def test_ring(self):
+        """Verify that examples/exp_configs/non_rl/ring.py is working."""
+        self.run_simulation(non_rl_ring)
 
     def test_bay_bridge(self):
-        """Verifies that examples/sumo/bay_bridge.py is working."""
-        # import the experiment variable from the example
-        exp = bay_bridge_example(render=False)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
-
-        # import the experiment variable from the example with inflows
-        exp = bay_bridge_example(render=False, use_inflows=True)
-
-        # run the experiment for a few time steps to ensure it doesn't fail
-        exp.run(1, 5)
+        """Verify that examples/exp_configs/non_rl/bay_bridge.py is working."""
+        # test without inflows and traffic lights
+        self.run_simulation(non_rl_bay_bridge)
 
-        # import the experiment variable from the example with traffic lights
-        exp = bay_bridge_example(render=False, use_traffic_lights=True)
+        # test with inflows
+        # FIXME
```

Comment:
Another issue for this. A more informative comment might also be a good idea here

Reply:
Yes. I already have a todo for opening PR for the remaining tests that have FIXME. I will push soon